inc/core/patterns/general-breadcrumb-nav.php

```diff
@@ -12,7 +12,7 @@
 return [
 	'title'      => __( 'Breadcrumb Navigation', 'bcgov_blocks_theme' ),
 	'categories' => [ 'bcgov-blocks-theme-general' ],
-	'content'    => '<!-- wp:group {"tagName":"section","style":{"spacing":{"padding":{"top":"1rem","right":"1rem","bottom":"1rem","left":"1rem"}}},"layout":{"inherit":false,"contentSize":"1440px"},"fontSize":"extra-small"} -->
-	<section class="wp-block-group has-extra-small-font-size"><!-- wp:aioseo/breadcrumbs /--></section>
+	'content'    => '<!-- wp:group {"tagName":"section"},"layout":{"inherit":false,"contentSize":"1440px"},"fontSize":"extra-small"} -->
+	<section class="wp-block-group has-extra-small-font-size" style="><!-- wp:aioseo/breadcrumbs /--></section>
```

Comment:
Did the block editor leave this as `style="` ?

Reply:
Brutal.. Will fix.